tfrobot/pkg/deployer/deployer.go

```diff
@@ -123,8 +123,16 @@ func deployNodeGroup(
 	vms []Vms,
 	sshKeys map[string]string,
 ) error {
+	var ygg bool
```

Comment:
this is only needed for `filterNodes` should we move it in the filtration function?